ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1019,20 +1019,14 @@ def test_expand(
 
 @st.composite
 def _concat_from_sequence_helper(draw):
-    shape = draw(helpers.get_shape(min_num_dims=1))
-    num_arrays = draw(
-        st.shared(helpers.ints(min_value=1, max_value=3), key="num_arrays")
-    )
-    dtype = draw(st.sampled_from(draw(helpers.get_dtypes("valid"))))
-    arrays = []
-    dtypes = [dtype for _ in range(num_arrays)]
-    for _ in range(num_arrays):
-        array = draw(helpers.array_values(dtype=dtype, shape=shape))
-        arrays.append(np.asarray(array, dtype=dtype))
-
-    axis = draw(
-        helpers.get_axis(shape=st.shared(helpers.get_shape(), key="value_shape"))
+    dtypes, arrays, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            num_arrays=helpers.ints(min_value=1, max_value=6),
+            ret_shape=True,
```

Comment:
About the 0 dimensional array error, I think you could add a `min_num_dims=1` here to avoid it.

Reply:
There is an error coming in the container function, I think it's because the testing function should be passing arrays of equal shapes, that's why.

it seems that the shapes of the arrays in the input_sequence parameter are not compatible for concatenation along the specified axis. The error message indicates that the shape of the first array in the input_sequence parameter is (1, 32, 512), while the shape of the second array is (1, 1, 768)

This is the error message:
```
cls = <class 'ivy.data_classes.container.container.Container'>, instance = 'c'

	def __instancecheck__(cls, instance):
    	"""Override for isinstance(instance, cls)."""
>   	return _abc_instancecheck(cls, instance)
E   	RecursionError: maximum recursion depth exceeded in comparison
E   	Falsifying example: test_concat_from_sequence(
E       	dtypes_arrays_axis=(['uint8'], [array([0], dtype=uint8)], 0),
E       	new_axis=0,
E       	ground_truth_backend='tensorflow',
E       	test_flags=FunctionTestFlags(
E           	num_positional_args=1,
E           	with_out=False,
E           	instance_method=True,
E           	test_gradients=False,
E           	test_compile=False,
E           	as_variable=[False],
E           	native_arrays=[False],
E           	container=[True],
E       	),
E       	fn_name='concat_from_sequence',
E       	backend_fw=<module 'ivy.functional.backends.torch' from '/workspaces/ivy/ivy/functional/backends/torch/__init__.py'>,
E       	on_device='cpu',
E   	)
E  	 
E   	You can reproduce this example by temporarily adding @reproduce_failure('6.68.2', b'AXicY2BgZGRAADCbEQAAQQAF') as a decorator on your test case

/usr/lib/python3.8/abc.py:98: RecursionError
!!! Recursion error detected, but an error occurred locating the origin of recursion.
  The following exception happened when comparing locals in the stack frame:
	IndexError: list index out of range
  Displaying first and last 10 stack frames out of 950.
--------------------------------------------------- Captured stdout call ---------------------------------------------------
  File "/workspaces/ivy/ivy/utils/exceptions.py", line 122, in new_fn
	return fn(*args, **kwargs)
  File "/workspaces/ivy/ivy/func_wrapper.py", line 655, in new_fn
	return fn(*args, **kwargs)
  File "/workspaces/ivy/ivy/func_wrapper.py", line 582, in new_fn
	return fn(*args, **kwargs)
  File "/workspaces/ivy/ivy/func_wrapper.py", line 261, in new_fn
	return fn(*new_args, **new_kwargs)
  File "/workspaces/ivy/ivy/func_wrapper.py", line 323, in new_fn
	ret = fn(*args, **kwargs)
  File "/workspaces/ivy/ivy/func_wrapper.py", line 177, in new_func
	return func(*args, **kwargs)
  File "/workspaces/ivy/ivy/functional/backends/torch/experimental/manipulation.py", line 288, in concat_from_sequence
	ret = torch.cat(input_sequence, dim=axis)
```

Reply:
I don't think  that's the reason. In the falsifying example you provided it seems that in this case only one array is being generated.

Reply:
I would suggest you use the debugger on that specific example. To run the falsifying case , you can copy-paste the `@reproduce_failure` part you get in the error message right after `@handle_test`.